src/Language/PureScript/TypeChecker/Kinds.hs

```diff
@@ -357,6 +358,22 @@ unifyKinds = unifyKindsWithFailure $ \w1 w2 ->
   throwError
     . errorMessage''' (fst . getAnnForType <$> [w1, w2])
     $ KindsDoNotUnify w1 w2
+  -- throwError
+  --   . errorMessage
+  --   $ KindsDoNotUnify w1 w2
```

Comment:
This wasn't meant to be left here, was it?

Reply:
Nope! 😄